src/app/app.component.ts

```diff
@@ -49,13 +51,19 @@ export class AppComponent implements OnInit {
     shareReplay(1),
   );
   loadingDb = true;
+  isMobileDevice = this.isMobile();
+  dropZoneLabel = this.isMobileDevice ?
+    'Select supported Files (HTMLZ & EPUB) to continue.' :
+    'Drop / select supported Files (HTMLZ & EPUB) or Directories (Right Click) containing them to continue.';
```

Comment:
```
'Drop or select a single file (.htmlz or .epub) or a folder that contains those files to continue';
```

Reply:
does this imply that the right click handler on the dropzone should be removed (and only drop and the icon will trigger it)? Otherwise removing the right click term from the description means no user will know how to do a directory selection from the dropzone without drag&drop